### PR TITLE
Update Android image classification example to use ONNX model zoo

### PR DESCRIPTION
**Pull request by edgchen1**
Motivation: Move away from downloading pre-converted ORT format models from Azure storage container.

### CODE REVIEW

File: mobile/examples/image_classification/android/prepare_models.requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+onnx==1.12.0
+onnxruntime==1.12.1
```
Comment (edgchen1) [inline mobile/examples/image_classification/android/prepare_models.requirements.txt:2]: if this PR is not merged before the ORT release, update to 1.13.

File: mobile/examples/image_classification/android/prepare_models.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        "https://raw.githubusercontent.com/pytorch/hub/master/imagenet_classes.txt",
+        args.output_dir / "imagenet_classes.txt",
+    )
+
+    # prepare models in temporary directory, then copy them to output directory
+    with tempfile.TemporaryDirectory(dir=args.output_dir) as temp_dir:
+        temp_dir = pathlib.Path(temp_dir)
+
+        for model_name, model_path_stem in [
+            ("MobileNet v2-1.0-int8", "mobilenetv2_int8"),
```
Comment (YUNQIUGUO) [inline mobile/examples/image_classification/android/prepare_models.py:59]: nit: Is it okay to use int8 mobilenet model? just noticed previously the model was an uint8 one.

Comment (edgchen1) [reply mobile/examples/image_classification/android/prepare_models.py:59]: it seems to work. I'm not sure how much of a difference it makes. @yufenglee any idea?
the ONNX model zoo only has an int8 quantized version.

Comment (yufenglee) [reply mobile/examples/image_classification/android/prepare_models.py:59]: I would recommend int8 model for mobile. Int8 will be faster than uint8 on arm64 without dot product.

File: mobile/examples/image_classification/android/prepare_models.requirements.txt (hunk 1 above)
Comment (edgchen1) [reply mobile/examples/image_classification/android/prepare_models.requirements.txt:2]: will do this in another PR

**Pull request merged**